src/renderer/gdi/gdirenderer.hpp

```diff
@@ -95,6 +100,7 @@ namespace Microsoft::Console::Render
         HFONT _hfont;
         HFONT _hfontItalic;
         TEXTMETRICW _tmFontMetrics;
+        FontResource _softFont;

```

Comment:
How much does this grow the GdiEngine by? I'm not terribly concerned about it, but it's just a thing to keep in mind.

Reply:
I think it's 80 bytes in the x64 build. There's a `std::vector` (24 bytes), two `til::size` fields (16 bytes each), a `size_t` and two resource handles (8 each). I'm guessing the x86 build is around half that. Obviously there's going to be more memory involved once you actually create a soft font, but that'll be freed again if you execute an `RIS` reset.